src/gluonts/dataset/rolling_dataset.py

```diff
@@ -100,6 +100,19 @@ def get_windows(self, window):
             yield window[: int(round(slice_idx))]
 
 
+def truncate_features(timeseries: dict, max_len):
```

Comment:
Maybe add complete type annotations (`max_len` and the return type?)

Reply:
True that, added it!